runtime/caml/memory.h

```diff
@@ -192,73 +193,50 @@ extern uintnat caml_use_huge_pages;
     (((x) + (Heap_page_size - 1)) & ~ (Heap_page_size - 1))
 #endif
 
-
-int caml_page_table_add(int kind, void * start, void * end);
-int caml_page_table_remove(int kind, void * start, void * end);
-int caml_page_table_initialize(mlsize_t bytesize);
-
 #ifdef DEBUG
 #define DEBUG_clear(result, wosize) do{ \
   uintnat caml__DEBUG_i; \
   for (caml__DEBUG_i = 0; caml__DEBUG_i < (wosize); ++ caml__DEBUG_i){ \
-    Field ((result), caml__DEBUG_i) = Debug_uninit_minor; \
+    Op_val (result)[caml__DEBUG_i] = Debug_uninit_minor; \
```

Comment:
Consider reverting.

Reply:
Fixed in https://github.com/ocaml-multicore/ocaml/commit/0c85f2822ec96beb15b6ecad5cd74a34d8ef54a9